plugins/explore-backend/src/service/router.ts

```diff
@@ -19,14 +19,15 @@ import { GetExploreToolsRequest } from '@backstage/plugin-explore-common';
 import express from 'express';
 import Router from 'express-promise-router';
 import { Logger } from 'winston';
-import { ExploreToolProvider } from '../tools';
+import { ExploreToolProvider, ProviderFactories } from '../tools';
 
 /**
  * @public
  */
 export interface RouterOptions {
   logger: Logger;
   toolProvider: ExploreToolProvider;
+  providerFactories?: ProviderFactories;
```

Comment:
This field is never read by anyone.

Reply:
Yeah, I think I did, `providerFactories` field is removed from `RouterOptions`